src/libsync/account.h

```diff
@@ -31,11 +31,12 @@
 #include <QPixmap>
 #endif
 
-#include "common/utility.h"
-#include <memory>
 #include "capabilities.h"
 #include "clientsideencryption.h"
+#include "common/utility.h"
 #include "syncfileitem.h"
+#include "theme.h"
```

Comment:
@claucambra please do not abuse with includes to headers as this will affect compilation speed, in the case of `QColor _serverColor` the solution would be to initialize it in the constructor and move `"theme.h"` into `.cpp` file.

Reply:
Good point, fixed this, thanks